src/webgpu/shader/util.ts

```diff
@@ -0,0 +1,42 @@
+export const PositiveInf = 0x7f800000;
+export const NegativeInf = 0xff800000;
```

Comment:
nit: kConstantName

nit: to be more descriptive, kPositiveInfF32AsInteger, or something like that?

nit: src/webgpu/util/math.ts would probably be a good place for these